### DISCUSSION TITLE
✔  Application for OWASP LAB project status - round 2.

### DISCUSSION THREAD

**Discussion by righettod** [oshp/oshp-tracking#7] [Ideas]
Hi leaders,

What do you think about starting the process to apply for the **LAB** status for the project?

![image](https://user-images.githubusercontent.com/1573775/153744883-8a76d384-019c-4935-bab0-26186afbd6e6.png)

Sources: 

* [Projects page](https://owasp.org/projects/)
* [Project Graduation Application](https://docs.google.com/forms/d/e/1FAIpQLScIhMp5DtPAJAIT_VTSmtwGjFjLyFYj9f_NfEgfYPjPNiUTbA/viewform)

OSHP exist since several years now and has proven to be stable and reliable in terms of provided content.

If you are OK, I can handle that point 👍 

Thanks in advance 😃

**Comment by righettod** [comment]
Committee feedback:

![image](https://user-images.githubusercontent.com/1573775/160227242-9a3f8aeb-406a-417b-90f9-dedc29b8f790.png)

My reply:

![image](https://user-images.githubusercontent.com/1573775/160227252-1cb5a260-9870-4964-9778-2aeccabf5919.png)

**Comment by righettod** [comment]
Please find below the result of the work to address the raised issues.

> Considered "Documentation" but points to various tools in `https://github.com/oshp`. Unclear what the main aspect is meant to be. 

The project objectives and goals were clarified and explicitly documented on the **Description** section of the main page of the web site:

![image](https://user-images.githubusercontent.com/1573775/168301091-7110c727-5be9-4df7-8ed7-e31e81a2257b.png)

> Should this just be a cheat sheet (except for all those tools...)

The project is more than a CS because we provide unified and consistent material regarding the HTTP security headers from technical survey on new headers, documentation/guidance on existing one, up to the recommended configuration and testing.

> Promotion request was submitted for the project website repo but not the code repo? 

The OWASP Secure Headers Project is composed by :
* The central [website](https://owasp.org/www-project-secure-headers/) that is the project's entry point.
*  All the **active projects** under the [OSHP GitHub organization](https://github.com/oshp/).

It's a global project focusing on HTTP security headers providing different elements regarding this topic.

> Project website has several informational tabs on HTTP headers but does not explain the tool found at `https://github.com/oshp/headers` at all. 

The status on the fact the this project was discontinued was added on the main page.

> GitHub main project had last update 9 months ago. Hard to tell if this should be considered "finished" or already on its way to "outdated". 

The status on the fact the this project was discontinued was added on the main page.

> Link `http://oshp.bsecteam.com/` mentioned on GitHub causes uBlockOrigin warning and redirects to `http://gwrtheyrn-rot.com/zcredirect?visitid=afc55dc0-a191-11ec-9d32-12b602a07ea5&type=js&browserWidth=1920&browserHeight=937&iframeDetected=false`. 

The affected URL was replaced by the one of the central [website](https://owasp.org/www-project-secure-headers/).

**Comment by righettod** [comment]
Application submitted for round 2 🤞

![image](https://user-images.githubusercontent.com/1573775/178263879-2014022f-2a40-4950-bc8b-c257b9831d2a.png)

**Comment by righettod** [comment]
Promotion accepted 💯 

![image](https://user-images.githubusercontent.com/1573775/186465966-0f460c55-f5f2-47f7-8ef9-524c56a066bc.png)

**Comment by riramar** [comment]
Amazing! Thanks a lot @righettod !!!
